include/deal.II/lac/la_parallel_block_vector.h

```diff
@@ -366,6 +366,12 @@ namespace LinearAlgebra
       bool
       has_ghost_elements() const;
 
+      /**
+       * Change the ghost state of this vector to @p ghosted.
```

Comment:
```suggestion
       * Change the ghost state of all blocks in this vector to @p ghosted.
```
Only if you think this addition improves the documentation :)